src/main/java/de/eldoria/semvertools/util/Integers.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (c) 2021 SemVerTools team and contributors
+ */
+
+package de.eldoria.semvertools.util;
+
+import org.jetbrains.annotations.ApiStatus;
+
+import java.util.Optional;
+
+/**
+ * A utility class for operations on integers.
+ */
+@ApiStatus.Internal
+public enum Integers {
+  ;
```

Comment:
He feels lonely

Reply:
The cost of misusing enums as utility classes...